CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/dragdrop/FileDropper.java

```diff
@@ -105,4 +82,42 @@ public void execute(final PluginGraphs graphs, final PluginInteraction interacti
 
         return null;
     }
+
+    @PluginInfo(pluginType = PluginType.IMPORT, tags = {"LOW LEVEL"})
```

Comment:
I'd go with `IMPORT` as this is adding data from an external file (via drag and drop) onto the graph.